### PR TITLE
881 Refine all the notebooks

### LINKED ISSUE #881: There are several minor issues in new updated notebooks
**Describe the bug**
We just had a very big update for all the notebooks which also brought some minor issues.
Reviewing in process, use this ticket to track:
1. Need to update the dataset paths to public dir instead of a personal home path.
2. Need to update the plot of CacheDataset notebook, used wrong data to plot.
3. Use torch.cuda to profile memory usage.
I will submit a PR to fix all the above issues after review.
4. Some random outputs are not "suitable" to show the transform results.

### PR DESCRIPTION
**Pull request by Nic-Ma**
Fixes #881 .

### Description
This PR resolved new added issues in notebooks.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh --codeformat --coverage`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: examples/notebooks/brats_segmentation_3d.ipynb [hunk 1]
```diff
@@ -77,11 +77,33 @@
...
    "metadata": {
     "tags": []
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "MONAI version: 0.1.0+318.ga2cd842\n",
```
Comment (wyli) [inline examples/notebooks/brats_segmentation_3d.ipynb:89]: it is still showing monai v0.1.0, could you run `git fetch --tags` and rerun this cell? might be a bug for the versioning

Comment (Nic-Ma) [reply examples/notebooks/brats_segmentation_3d.ipynb:89]: Hi @wyli ,

I tried to `git fetch --tags`, it's still MONAI v0.1+.
I think maybe this is some bug in versioning tool, could you please help file a ticket for it?

Thanks.

Comment (wyli) [reply examples/notebooks/brats_segmentation_3d.ipynb:89]: couldn't replicate this issue from my side

Comment (Nic-Ma) [reply examples/notebooks/brats_segmentation_3d.ipynb:89]: What did you use to test? repo or pip wheel? I used `pip install monai` directly.
Thanks.

Comment (Nic-Ma) [reply examples/notebooks/brats_segmentation_3d.ipynb:89]: I opened a new docker container, issue resolved.
Seems some cache in my previous docker container caused this issue.
Thanks.

Comment (Nic-Ma) [reply examples/notebooks/brats_segmentation_3d.ipynb]: Hi @wyli , I already updated all the notebooks with latest MONAI and PyTorch version.
Thanks for raising this issue.

Comment (wyli) [reply examples/notebooks/brats_segmentation_3d.ipynb]: thanks, I think we need to clarify the steps to run the notebook examples (as a part of https://github.com/Project-MONAI/MONAI/issues/872). probably we don't need `pip install monai` after git cloning the source code

**Comment by Nic-Ma** [comment]
> could you clear up the output logs, this pr currently adds 30k+ new lines

Hi @wyli ,

Thanks for your review, I have removed the logs now.

**Comment by wyli** [comment]
Hi @Nic-Ma, I've migrated this branch to the tutorial repo with all the relevant git history, please have a look https://github.com/Project-MONAI/Tutorials and file issues there if needed. I'm closing this PR now.

**Comment by Nic-Ma** [comment]
Sounds good.
Thanks.

**Comment by wyli** [comment]
> Sounds good.
> Thanks.

ok thanks, the repo is now public

**Pull request closed without merging**